Cargo.toml

```diff
@@ -33,18 +33,6 @@ members = [
     "view/desktop/bin",
 ]
 
-[profile.release]
-# This makes the compiled code faster and smaller, but it makes 
-# compiling slower, so it's only enabled in release mode.
-lto = false
-opt-level = 3
-panic = "abort"
-codegen-units = 1
-
-[profile.dev.package.sqlx-macros]
-opt-level = 3
-
-
```

Comment:
There is no need to delete this, these optimizations will be needed when we build the binary, so let's just comment it out for now

Reply:
Restored and commented out.